src/components/SubscribeEmail/SubscribeEmail.js

```diff
@@ -1,11 +1,53 @@
 import React from 'react';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import { postData } from 'components/LoginRegistration/LoginCheck';
 import classes from './SubscribeEmail.module.css';
 
 function SubscribeEmail() {
+  const emailSubcribe = React.useRef();
+  const [show, setShow] = React.useState(false);
+  const [subscribeErrors, setSubscribeError] = React.useState();
+
+  const subcribeEmailCheck = localStorage.getItem('subcribeEmailRemember');
+
+  const subcribe = async () => {
+    const subscribeResponse = await postData(
+      'https://vnguyen.xyz/huy/day17/apis/index.php?type=subscribes',
+      {
+        email: emailSubcribe.current.value,
+      }
+    );
+    const { message, errors } = subscribeResponse;
+    const errorFields = errors ? JSON.parse(errors)?.fields : message;
+    const [_, firstError] = Object.entries(errorFields).find(
+      ([_key, error]) => {
+        if (error) {
+          return error;
+        }
+        return [];
+      }
+    );
+    const errorFormat = firstError ? Object.values(firstError)?.[0] : undefined;
+    if (errorFormat !== 'T') {
+      setSubscribeError(`${errorFormat}`);
+      setShow(true);
+      return;
+    }
+    setSubscribeError(`Thank you for Subcribing`);
+    localStorage.setItem('subcribeEmailRemember', true);
+    setShow(true);
+  };
+  const handleClose = () => setShow(false);
+
   return (
-    <div className={classes.subscribe_email}>
+    <div
+      className={`${classes.subscribe_email} ${
+        subcribeEmailCheck === 'true' ? 'd-none' : ''
```

Comment:
Do not use a hidden class to hide a component, remember our component can be hidden by condition.